WcaOnRails/app/models/competition.rb

```diff
@@ -2011,4 +2011,30 @@ def dues_per_competitor_in_usd
     dues = DuesCalculator.dues_per_competitor_in_usd(self.country_iso2, self.base_entry_fee_lowest_denomination.to_i, self.currency_code)
     dues.present? ? dues : 0
   end
+
+  private def xero_dues_payer
+    if self.country.wfc_dues_redirect.present?
+      self.country.wfc_dues_redirect.redirect_to
+    elsif self.organizers.any? { |organizer| organizer.wfc_dues_redirect.present? }
+      self.organizers.find { |organizer| organizer.wfc_dues_redirect.present? }.wfc_dues_redirect.redirect_to
+    end
+  end
+
+  def dues_payer_name
+    dues_payer = xero_dues_payer || staff_delegates.min_by(&:name)
+    dues_payer&.name
+  end
+
+  def dues_payer_email
+    dues_payer = xero_dues_payer || staff_delegates.min_by(&:name)
```

Comment:
This duplicates the `staff_delegates.min_by` from above.
Please consider refactoring this into a separate `private def delegate_dues_payer` email, which can then include a comment about why sorting by name makes sense (or why WFC decided that it makes sense)